lib/core/tests/text.rs

```diff
@@ -4,8 +4,8 @@
 use basegl::display::world::Add;
 use basegl::display::world::Workspace;
 use basegl::display::world::WorkspaceID;
+use basegl::display::world::WorldData;
```

Comment:
Remove unused imports. BTW Why lint is not failing here? Perhaps due to restriction to wasm32 target.

We should consider use flag in cargo similar to -Wall in gcc

Reply:
Oh, this is bad that its not catche automatically. Would you be able to improve the auto linter config?

Reply:
Ok, I try find on some occasion